### PR TITLE
feat: add note, edit-note

### PR DESCRIPTION
**Pull request by BelideNikhil**
+ add new note
+ edit note

### CODE REVIEW

File: src/Components/EditNote/EditNote.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100vh;
+}
+.edit-note-content {
+    max-width: 30rem;
+    background-color: var(--clr-whitesmoke);
+    position: absolute;
```
Comment (msdeshmukh009) [inline src/Components/EditNote/EditNote.css]: You don't need this.

File: src/Components/NoteCard/NoteCard.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+.note .note-title {
+    font-size: 1.1rem;
+    font-weight: var(--fw--semi-bold-500);
+}
+.note-text {
+    word-break: break-all;
+}
+.note-pin {
+    width: 24px;
+    height: 24px;
```
Comment (msdeshmukh009) [inline src/Components/NoteCard/NoteCard.css:31]: You can use rem here.

**Review by msdeshmukh009 — APPROVED** [review]
This is looking good. The note input is really working seamlessly. Good work. Please merge it.

**Comment by Chakravarthi0** [comment]
UI looks good.
good to merge.

**Review by Chakravarthi0 — APPROVED** [review]
UI looks good.
good to merge.

**Pull request merged**